assets/js/main.js

```diff
@@ -0,0 +1,17 @@
+const cssEditor = document.getElementById("css-editor");
+const head = document.head;
+const style = document.createElement("style");
+
+style.type = "text/css";
+head.insertAdjacentElement("beforeend", style);
+
+const editorKeyupEvent = () => {
+  editorValue = cssEditor.value;
+  try {
```

Comment:
I like this... maybe this was what I was missing in my logic to make it work. Very nice use of the old try and catch !